src/styles/global.css

```diff
@@ -148,7 +148,9 @@
     /* Complementary Set */
     --attention-outline: var(--attention-light);
 
-    --error-light: var(--error-light);
+    --error: var(--red-500);
+    --error-light: var(--red-100);
+    --error-outline: var(--error);
```

Comment:
Curious the reason for the `-outline` token variants?

Reply:
Hmm good call, we are not using it, right? I guess the idea was to use this color as the `border` color. I missed that.